doc/source/glossary.rst

```diff
@@ -330,6 +542,32 @@ Glossary
        >>> x[:, 1]
        array([2, 4])
 
+   stride
+
+      Physical memory is one-dimensional; ``stride`` maps an index in an
+      N-dimensional ndarray to an address in memory. For an N-dimensional
+      array, stride is an N-element tuple; advancing from index ``i`` to index
```

Comment:
```suggestion
      array, a tuple of N strides is needed; advancing from index ``i`` to index
```

Reply:
array, its ``stride`` attribute is an N-element tuple ...

Reply:
The attribute is called `strides` with an `s`